### PR TITLE
SPECIES NUKING 2023 EXTRA: Makes skin tones a trait instead of a species variable

### PR DESCRIPTION
**Pull request by ChungusGamer666**
## About The Pull Request

very similar to https://github.com/tgstation/tgstation/pull/76297, much simpler though
removes the uses_skintones variable for similar reasons

## Why It's Good For The Game

Species variables are less useful and modular than traits. 
If for some reason we ever add a mechanic that for some reason makes your character use skin tones instead of whatever their original color is, it will be much simpler to be handled.

## Changelog

not player facing

### CODE REVIEW

**Comment by ChungusGamer666** [comment]
what the fuck how did they fail

File: code/modules/mob/living/carbon/human/species_types/dullahan.dm [hunk 1]
```diff
@@ -3,12 +3,12 @@
 	id = SPECIES_DULLAHAN
 	examine_limb_id = SPECIES_HUMAN
 	inherent_traits = list(
+		TRAIT_SKINTONES,
 		TRAIT_NOBREATH,
 		TRAIT_NOHUNGER,
```
Comment (san7890) [inline code/modules/mob/living/carbon/human/species_types/dullahan.dm:8]: alphabetize

File: code/modules/mob/living/carbon/human/species_types/felinid.dm [hunk 2]
```diff
@@ -9,7 +9,11 @@
 	external_organs = list(
 		/obj/item/organ/external/tail/cat = "Cat",
 	)
-	inherent_traits = list(TRAIT_CAN_USE_FLIGHT_POTION, TRAIT_HATED_BY_DOGS)
+	inherent_traits = list(
+		TRAIT_SKINTONES,
+		TRAIT_CAN_USE_FLIGHT_POTION, 
+		TRAIT_HATED_BY_DOGS,
```
Comment (san7890) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:15]: ditto alpha

File: code/modules/mob/living/carbon/human/species_types/humans.dm [hunk 3]
```diff
@@ -2,10 +2,10 @@
 	name = "\improper Human"
 	id = SPECIES_HUMAN
 	inherent_traits = list(
+		TRAIT_SKINTONES,
 		TRAIT_CAN_USE_FLIGHT_POTION,
```
Comment (san7890) [inline code/modules/mob/living/carbon/human/species_types/humans.dm:6]: ditto alpha

File: code/modules/mob/living/carbon/human/species_types/vampire.dm [hunk 4]
```diff
@@ -9,6 +9,7 @@
 	id = SPECIES_VAMPIRE
 	examine_limb_id = SPECIES_HUMAN
 	inherent_traits = list(
+		TRAIT_SKINTONES,
 		TRAIT_BLOOD_CLANS,
 		TRAIT_NOBREATH,
```
Comment (san7890) [inline code/modules/mob/living/carbon/human/species_types/vampire.dm:14]: ditto alphabetize

File: code/__DEFINES/traits.dm [hunk 5]
```diff
@@ -230,6 +230,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_BLOOD_CLANS "blood_clans"
 /// Species with this trait have markings (this SUCKS, remove this later in favor of bodypart overlays)
 #define TRAIT_HAS_MARKINGS "has_markings"
+/// Species with this trait use skin tones for coloration
+#define TRAIT_SKINTONES "skintones"
```
Comment (san7890) [inline code/__DEFINES/traits.dm:234]:
```suggestion
#define TRAIT_USES_SKINTONES "skintones"
```

i think this is a bit more verbose? kinda fits in with the whole variable it was replacing. shouldn't be too hard to find and replace. if you think it's too nitty feel free to ignore but i think it's neat

**Review by san7890 — CHANGES_REQUESTED** [review]
i'm going to trust someone who is much more knowledgable in matters species to merge this PR but i may as well get my nits in

File: code/__DEFINES/traits.dm (hunk 5 above)
Comment (ChungusGamer666) [reply code/__DEFINES/traits.dm:234]: Eh, I like my variables more verbose. I'll do this.

**Comment by ChungusGamer666** [comment]
> i'm going to trust someone who is much more knowledgable in matters species to merge this PR but i may as well get my nits in

ayo @Time-Green review dis shit my homie

**Review by Time-Green — APPROVED** [review]

**Pull request merged**